### ISSUE TITLE
'Install susper' button not working for Chrome browser.

### ISSUE THREAD

**Issue by harshit98** [fossasia/susper.com#314]
'Install Susper' button clicking on it is not working for the Chrome browser. It should work fine, as it does for Firefox browser. 

Chrome browser -

![screenshot from 2017-05-21 19-22-15](https://cloud.githubusercontent.com/assets/22245418/26284523/e3691738-3e5a-11e7-874f-1926d1f28276.png)

**Comment by harshit98** [comment]
@nikhilrayaprolu @Marauderer97 I was working on this issue, and found that the `AddSearchProvider()` and `window.external.IsSearchProviderInstalled()` works only with some versions of Firefox and earlier they worked with Chrome as well. Is there any alternative idea we can implement ?

**Comment by nikhilrayaprolu** [comment]
http://blog.unto.net/add-opensearch-to-your-site-in-five-minutes.html

**Comment by harshit98** [comment]
I have done the changes in xml file. Kindly explain, what will happen? I'm have not seen any changes even after updating the xml file.

**Comment by harshit98** [comment]
@nikhilrayaprolu @Marauderer97 I have the done work on this issue following the link you have provided. It would be great if I can get suggestions from your side. What I was thinking for this issue : 
- For the Chrome browser, I saw the feature implemented by DuckDuckGo. Check here - https://duckduckgo.com/install?t=hc . Chrome has some different feature to add search engines. So, we can't implement the box like feature which will not work with Chrome. We can either provide a link in tutorial page to add susper as search engine in Chrome following steps as DuckDuckGo has done. 
- For Firefox, the box works fine. And following the opensearch link, I have also added the feature which will automatically detect the search engine in Firefox like this -

![screenshot from 2017-05-26 00-02-56](https://cloud.githubusercontent.com/assets/22245418/26466021/e153a55a-41aa-11e7-932d-3fbf4d143646.png)

So should we remove the default-box from Chrome and keep it for Firefox ? Or remove the box for both browsers ? At present, the functions `AddSearchProvider()` and `window.external.IsSearchProviderInstalled()` does not work for Chrome browser. For chrome, there were previously `IsDefaultSearchProvider()` and `SetDefaultSearchProvider` functions which has been now removed.
